tests/Functional/Platform/RenameColumnTest.php

```diff
@@ -95,31 +94,4 @@ public static function columnNameProvider(): iterable
         yield ['C1', 'c1_x'];
         yield ['importantColumn', 'veryImportantColumn'];
     }
-
-    /** @throws Exception */
-    public function testRenameColumToQuoted(): void
```

Comment:
This test was introduced in https://github.com/doctrine/dbal/pull/6602 and is no longer necessary. All new names are quoted by default, so this case is covered by existing tests.